platform/ios/uitest/MapViewTests.m

```diff
@@ -274,6 +274,21 @@ - (void)testRotate {
                                 @"rotating map should change angle");
 }
 
+- (void)testRotateCounterclockwise {
+    CLLocationDirection startAngle = tester.mapView.direction;
+    
+    XCTAssertNotEqual(startAngle,
+                      -45,
+                      @"start angle must not be destination angle");
+    
+    [tester.mapView twoFingerRotateAtPoint:tester.mapView.center angle:-45];
+    
+    XCTAssertGreaterThanOrEqual(fabs(tester.mapView.direction - startAngle),
+                                -20,
```

Comment:
The absolute value of one number will always be greater than or equal to another, negative number.